Makefile

```diff
@@ -172,10 +178,32 @@ image-build:
 		$(PUSH) \
 		$(IMAGE_BUILD_EXTRA_OPTS) ./
 
+.PHONY: e2e-docker
+e2e-docker:
+	$(IMAGE_BUILD_CMD) -t $(IMAGE_TAG) \
+		--build-arg BASE_IMAGE=$(BASE_IMAGE) \
+		--build-arg BUILDER_IMAGE=$(BUILDER_IMAGE) \
+		--load \
+		$(PUSH) \
+		$(IMAGE_BUILD_EXTRA_OPTS) ./
+
 .PHONY: image-push
 image-push: PUSH=--push
 image-push: image-build
 
+.PHONY: image-load
+image-load:
+	kind load docker-image $(IMAGE_TAG)
+
+## Create Kind Cluster for E2E
+.PHONY: kind-create
+kind-create:
+	kind create cluster
```

Comment:
It would be better to specify the K8s version for the kind cluster.

Reply:
Would an environment variable in the make file be sufficient?

Reply:
We can specify node image by the `--image` flag as shown the below.

```shell
Creates a local Kubernetes cluster using Docker container 'nodes'

Usage:
  kind create cluster [flags]

Flags:
      --config string       path to a kind config file
  -h, --help                help for cluster
      --image string        node docker image to use for booting the cluster
```

node images: https://hub.docker.com/r/kindest/node/tags

Reply:
Yea  I was wondering about parameterizing the images for future e2e tests with different kubernetes versions.

Reply:
Added variable for allowing new versions of kubernetes versions.